sql/core/benchmarks/MiscBenchmark-results.txt

```diff
@@ -2,119 +2,126 @@
-collect limit 2 millions                       581 /  614          1.8         554.4       0.5X
+collect limit:                            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+collect limit 1 million                             341            348           5          3.1         325.6       1.0X
+collect limit 2 millions                            654            665           9          1.6         623.8       0.5X
 
 
 ================================================================================================
 generate explode
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 1.8.0_181-b13 on Linux 3.10.0-862.3.2.el7.x86_64
+OpenJDK 64-Bit Server VM 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03 on Linux 4.15.0-1021-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
-generate explode array:                  Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
-------------------------------------------------------------------------------------------------
-generate explode array wholestage off       15211 / 15368          1.1         906.6       1.0X
-generate explode array wholestage on        10761 / 10776          1.6         641.4       1.4X
+generate explode array:                   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+generate explode array wholestage off             15383          15663         395          1.1         916.9       1.0X
+generate explode array wholestage on              10593          10638          40          1.6         631.4       1.5X
 
-OpenJDK 64-Bit Server VM 1.8.0_181-b13 on Linux 3.10.0-862.3.2.el7.x86_64
+OpenJDK 64-Bit Server VM 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03 on Linux 4.15.0-1021-aws
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
-generate explode map:                    Best/Avg Time(ms)    Rate(M/s)   Per Row(ns)   Relative
-------------------------------------------------------------------------------------------------
-generate explode map wholestage off         22128 / 22578          0.8        1318.9       1.0X
-generate explode map wholestage on          16421 / 16520          1.0         978.8       1.3X
+generate explode map:                     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+generate explode map wholestage off               50364          50656         413          0.3        3001.9       1.0X
+generate explode map wholestage on                43484          43833         313          0.4        2591.9       1.2X
```

Comment:
This looks regression from last committed benchmark results. I don't think this PR can cause it. So to verify it, I ran this benchmark on current master branch:

```
[info] OpenJDK 64-Bit Server VM 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03 on Linux 4.15.0-1021-aws
[info] Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
[info] generate explode array:                   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------------------------------
[info] generate explode array wholestage off             15893          15991         138          1.1         947.3       1.0X
[info] generate explode array wholestage on              10798          10833          36          1.6         643.6       1.5X
[info] 
[info] Running benchmark: generate explode map
[info]   Running case: generate explode map wholestage off
[info]   Stopped after 2 iterations, 102915 ms
[info]   Running case: generate explode map wholestage on
[info]   Stopped after 5 iterations, 216755 ms
[info] 
[info] OpenJDK 64-Bit Server VM 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03 on Linux 4.15.0-1021-aws
[info] Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
[info] generate explode map:                     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
[info] ------------------------------------------------------------------------------------------------------------------------
[info] generate explode map wholestage off               50273          51458        1676          0.3        2996.5       1.0X
[info] generate explode map wholestage on                43113          43351         278          0.4        2569.8       1.2X
[info] 
```

The all benchmark result can be found at https://gist.github.com/viirya/c850b49e364e8e6aaa6271ae9e3343fc

Reply:
I open https://issues.apache.org/jira/browse/SPARK-27779 to track it.